server/sonar-server/src/main/java/org/sonar/server/ce/ws/CancelAllAction.java

```diff
@@ -48,7 +47,7 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request wsRequest, Response wsResponse) {
-    userSession.checkPermission(UserRole.ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?